helm/Chart.yaml

```diff
@@ -1,10 +1,13 @@
 apiVersion: v2
 name: zaakafhandelcomponent
 description: A Helm chart for installing Zaakafhandelcomponent
-version: 0.3.0
+version: 0.4.0
 appVersion: "latest"
 dependencies:
   - name: opentelemetry-collector
-    repository: https://open-telemetry.github.io/opentelemetry-helm-charts
-    condition: open-telemetry-collector.enabled
     version: 0.55.2
+    repository: https://open-telemetry.github.io/opentelemetry-helm-charts
+    condition: opentelemetry-collector.enabled
+  - name: solr
```

Comment:
solr helm chart version 9.1.3 uses the solr 9.6.0 image, currently in use at our test environment is 9.5.0

Reply:
Yes, but we do want to upgrade to Solr 9.6.0 of course. :-) What exactly is your point here @infojohn ?

Reply:
my point is that we haven't tested with that version yet